Octokit/Octokit.csproj

```diff
@@ -1,50 +1,50 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <Description>An async-based GitHub API client library for .NET and .NET Core</Description>
-    <AssemblyTitle>Octokit</AssemblyTitle>
-    <Authors>GitHub</Authors>
-    <Version>0.0.0-dev</Version>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <AssemblyName>Octokit</AssemblyName>
-    <PackageId>Octokit</PackageId>
-    <DebugType>embedded</DebugType>
-    <RepositoryUrl>https://github.com/octokit/octokit.net</RepositoryUrl>
-    <PackageProjectUrl>https://github.com/octokit/octokit.net</PackageProjectUrl>
-    <PackageIconUrl>https://f.cloud.github.com/assets/19977/1510987/64af2b26-4a9d-11e3-89fc-96a185171c75.png</PackageIconUrl>
-    <PackageIcon>octokit.png</PackageIcon>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageTags>GitHub API Octokit linqpad-samples dotnetcore</PackageTags>
-    <Copyright>Copyright GitHub 2017</Copyright>
-  </PropertyGroup>
+	<PropertyGroup>
```

Comment:
Hey @jefflill, Thanks for this change! ❤️ .  We use spaces in this project - they need to be updated in this file. Would you mind either:
1. Updating this to reflect what I mentioned above and pushing the update to this PR or...
3. Setting the fork to give [maintainers write access](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) and I'd be glad to make the change